sdcm/cluster.py

```diff
@@ -494,6 +494,12 @@ def _register_grafana_json(json_filename, url):
                         'scylla-dash.json': 'dashboards/db',
                         'scylla-dash-per-server.json': 'dashboards/db',
                         'scylla-dash-io-per-server.json': 'dashboards/db',
+                        'scylla-dash.1.5.json': 'dashboards/db',
+                        'scylla-dash-per-server.1.5.json': 'dashboards/db',
+                        'scylla-dash-io-per-server.1.5.json': 'dashboards/db',
+                        'scylla-dash.1.6.json': 'dashboards/db',
+                        'scylla-dash-per-server.1.6.json': 'dashboards/db',
+                        'scylla-dash-io-per-server.1.6.json': 'dashboards/db',
```

Comment:
This means that every version should get all jsons configured. Is that what we want though?